kernel/arch/dreamcast/include/dc/maple/keyboard.h

```diff
@@ -297,7 +299,7 @@ typedef struct kbd_state {
     uint32 key_queue[KBD_QUEUE_SIZE];
     int queue_tail;                     /**< \brief Key queue tail. */
     int queue_head;                     /**< \brief Key queue head. */
-    int queue_len;                      /**< \brief Current length of queue. */
+    atomic_int queue_len;               /**< \brief Current length of queue. */
```

Comment:
Shouldn't this really be atomic_size_t?

Reply:
It was an `int`, not a `size_t`. If I change it to a `atomic_size_t` then I'm introducing a change that's completely unrelated to the bug being fixed.

Reply:
You're right. I didn't even realize the damn thing started out as an `int`... Oops.